nipype/pipeline/engine/utils.py

```diff
@@ -192,37 +192,37 @@ def write_report(node, report_type=None, is_mapnode=False):
         'prev_wd': getattr(result.runtime, 'prevcwd', '<not-set>'),
     }
 
-    if hasattr(result.runtime, 'cmdline'):
+    if result.runtime.cmdline:
```

Comment:
Should these all be `is not None`, or are empty strings/dicts/zeros meant to be treated the same?

Reply:
I'll change the `cmdline`, `mem_peak_gb`, and `cpu_percent` 👍